artifactory/commands/dotnet/dotnetcorecli.go

```diff
@@ -20,27 +15,5 @@ func NewDotnetCoreCliCommand() *DotnetCoreCliCommand {
 }
 
 func (dccc *DotnetCoreCliCommand) Run() (err error) {
-	dccc.useNugetAddSource, err = isDotnetVersionAboveMin()
```

Comment:
What is the reason the min version check was removed? Was this check move to build-info-go?

Reply:
It was a check for the minimal supported version of the `nuget add source` command. However, the usage of this command was previously removed because it does not support configuring the protocol version.

Reply:
Got it. Thanks.